pom.xml

```diff
@@ -135,6 +138,16 @@
     </dependency>
   </dependencies>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.jenkins.plugins</groupId>
+        <artifactId>commons-text-api</artifactId>
+        <version>1.10.0-27.vb_fa_3896786a_7</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
```

Comment:
This should be picked up by the BOM but this is a chicken egg problem?

Reply:
yes